package.json

```diff
@@ -148,7 +148,7 @@
     "webpack": "^4.46.0",
     "webpack-bundle-analyzer": "^3.9.0",
     "webpack-cli": "^3.3.12",
-    "webpack-dev-server": "^3.11.3",
```

Comment:
webpack-relaterte oppgraderinger har vært pain in the a.. i det siste. Anbefaler derfor å beholde denne linja, ellers vil ikke `npm start` starte programmet.